projects/storefrontapp-e2e-cypress/package.json

```diff
@@ -7,9 +7,9 @@
   "scripts": {
     "cy:open": "cypress open",
     "cy:run": "cypress run",
-    "cy:run:ci:1905": "cypress run --config-file cypress.ci.1905.json --record --key $CYPRESS_KEY --tag \"1905,all\" --spec \"cypress/integration/!(vendor|b2b)/**/*.e2e-spec.ts\"",
```

Comment:
Why we dropped the tags?

Reply:
We are not using this target. I can place it back, no worries.